docs/inventory.md

```diff
@@ -0,0 +1,246 @@
+# NixOS project resource inventory
+
+This is the current list of hardware and services that everyone has access to.
+
+# Accounts
+## GitHub
+
+owner: Eelco, Rob, Domen, Garbas
+
+## Domains
+
+owner: Eelco
+* nixos.org - https://www.uniteddomains.com/
+
+## DNS
+
+owner: Eelco
+Managed manually be Eelco via the United Domains web UI
+
+## AWS account
+
+owner:  Infor
+alias: lb-nixos
+access: Rob and Eelco
+
+## Packet.net
+
+owner: Graham
+
+## Hetzner Cloud
+
+owner: Graham
+(for ofborg)
+
+## IRC logging bot
+
+owner: SamuelDR
+https://logs.nix.samueldr.com/nixos/
+
+nick:
+
+    {`-`}_
+## Channels.nix.gsc.io
+
+owner: Graham
+
+Tracks all the channel updates
+
+Gives you a graph when all the channels where updated and history.
+
+To help bisect things.
+
+hosted on Packet (personal account)
+
+## search.nix.gsc.io
+
+owner: Graham
+
+Searches all of the nixos project repositories with editor integration.
+
+Useful for spelunking.
+
+hosted on Packet (personal account)
+
+## events.nix.gsc.io
+
+owner: Graham
+
+rabbit MQ instance which can send you all the NixOS organization activity.
+
+hosted on Packet (personal account)
+
+## nix.ci
+
+owner: Graham
+
+ofborg instance and logs
+
+hosted on Packet.
+
+## howoldis.herokuapp.com
+
+owner: madjar/howoldis
+url: https://howoldis.herokuapp.com/
+repo: http://github.com/madjar/howoldis
+
+Domen might have access
+
+Tells you how old channels are and some doc
+
+## arch64 community builder
+
+owner: Graham
+access: community members that have asked access to it
+host: Packet
+
+lots of cores to build for the aarch64 platform
+
+## nixcon2017.org
+
+owner: Christine?
+
+
+## nixcon2018.org
+
+owner: zimbatm
+
+
+## YouTube channel for recorded talks
+
+owner: Cillian?
+url: https://www.youtube.com/channel/UCjqkNrQ8F3OhKSCfCgagWLg
+
+
+## NixOS community wiki + bot
+
+owner: samueldr
```

Comment:
While I have been marked as *maintainer* for [nix-community/wiki](https://github.com/nix-community/wiki), the wiki and the IRC bot is maintained by @fadenb. The [NixOS_Wiki:About](https://nixos.wiki/wiki/NixOS_Wiki:About) may be a relevant link for this section.

Reply:
thanks again! fixed